experimenter/Dockerfile

```diff
@@ -179,9 +179,11 @@ RUN apt-get --no-install-recommends install -y apt-utils ca-certificates postgre
 COPY --from=dev /usr/local/bin/ /usr/local/bin/
 COPY --from=dev /usr/local/lib/python3.11/site-packages/ /usr/local/lib/python3.11/site-packages/
 COPY --from=dev /experimenter/bin/ /experimenter/bin/
+COPY --from=dev /application-services/megazord/ /application-services/megazord/
 COPY --from=file-loader /experimenter/manage.py /experimenter/manage.py
 COPY --from=file-loader /experimenter/experimenter/ /experimenter/experimenter/
 COPY --from=ui /experimenter/experimenter/legacy/legacy-ui/assets/ /experimenter/experimenter/legacy/legacy-ui/assets/
 COPY --from=ui /experimenter/experimenter/nimbus-ui/build/ /experimenter/experimenter/nimbus-ui/build/
 COPY --from=ui /experimenter/experimenter/theme/static/ /experimenter/experimenter/theme/static/
 COPY --from=ui /experimenter/experimenter/static/scripts/ /experimenter/experimenter/static/scripts/
+ENV PYTHONPATH=$PYTHONPATH:/application-services/megazord
```

Comment:
Heyyyyyy look at that.  Yeah that's a tricky little bit of trickiness, but this looks right to me.  Good job @eliserichards .  You know what would've helped with this is having an integration test that stresses this.  Then we would've found it before we made the changes.  We should make sure we get an integration test that hits this soon 🙏 thnx @eliserichards this is great 🎉 🎉 🎉